certified-connectors/Tribal/apiDefinition.swagger.json

```diff
@@ -3303,6 +3295,82 @@
           }
         }
       }
+    },
+    "/": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@mszy-tribal it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

Reply:
I have added the known responses